modin/config/__main__.py

```diff
@@ -71,7 +71,7 @@ def export_config_help(filename: str):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "-export_path",
+        "--export_path",
```

Comment:
```suggestion
        "--export-path",
```
plus note that you should change the calling line in `conf.py` now, otherwise the variables' table won't be regenerated properly

Reply:
In the `conf.py`, `export_config_help` called directly (when we used separate process to call `python -m modin.config --export-path` file with configs didn't generated during build on the readthedocs website for some reason, so function direct call was used), so it shouldn't affect docs generating process.